src/scripts/install-chromedriver.sh

```diff
@@ -153,36 +157,20 @@ if [[ $CHROME_RELEASE -lt 115 ]]; then
     --output chromedriver_$PLATFORM.zip \
     "http://chromedriver.storage.googleapis.com/$CHROMEDRIVER_VERSION/chromedriver_$PLATFORM.zip"
 else 
-  MATCHING_CHROMEDRIVER_URL_RESPONSE=$(curl -s -o /dev/null -w "%{http_code}" 'https://edgedl.me.gvt1.com/edgedl/chrome/chrome-for-testing/$CHROMEDRIVER_VERSION/linux64/chromedriver-linux64.zip')
-  echo $MATCHING_CHROMEDRIVER_URL_RESPONSE
-  if [[ $MATCHING_CHROMEDRIVER_URL_RESPONSE == 404 ]]; then
-    echo "Matching Chrome Driver Version 404'd, falling back to first matching major version."
+  MATCHING_CHROMEDRIVER_DOWNLOAD_URL=$(curl https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json | jq --arg chromeDriverVersion $CHROMEDRIVER_VERSION --arg platform $PLATFORM '.versions.[] | select(.version=="\($chromeDriverVersion)") | .downloads.chromedriver.[] | select(.platform=="\($platform)") | .url')
```

Comment:
I modified and used this script in isolation, as a `run` step, to get to a usable CI state. Thanks for writing it! ❤️ 

- I printed the jq -V at the beginning of my script, and I saw that I had 1.6 in CI.
- jq 1.6 supports `--raw-output`, which lets us output the URL value sans quotes, which was good for later. There was a place elsewhere in the code where `sed` was used to get rid of quotes, so I got a brainwave to use `--raw-output`.
- The syntax `.versions.[]` (note the dot before `[]`) was acting up in CI, not locally. Seems to have had to do with jq versions, according to other people online. I switched to the narrower form.
- `shellcheck` was good to use, to find not-quoted values that needed extra quoting.

I think this diff has those ideas:

```suggestion
  MATCHING_CHROMEDRIVER_DOWNLOAD_URL=$(curl --silent https://googlechromelabs.github.io/chrome-for-testing/known-good-versions-with-downloads.json | jq --raw-output --arg chromeDriverVersion $CHROMEDRIVER_VERSION --arg platform $PLATFORM '.versions[] | select(.version=="\($chromeDriverVersion)") | .downloads.chromedriver[] | select(.platform=="\($platform)") | .url')
```